common/lib/common-utils/src/index.ts

```diff
@@ -9,23 +9,39 @@
  * @packageDocumentation
  */
 
-export * from "./assert";
-export * from "./indexNode";
-export * from "./base64Encoding";
-export * from "./disposal";
-export * from "./eventForwarder";
-export * from "./heap";
-export * from "./logger";
-export * from "./promiseCache";
-export * from "./promises";
-export * from "./rangeTracker";
-export * from "./rateLimiter";
-export * from "./safeParser";
-export * from "./timer";
-export * from "./trace";
-export * from "./typedEventEmitter";
-export * from "./unreachable";
-export * from "./lazy";
-export * from "./performanceIsomorphic";
-export * from "./delay";
-export * from "./bufferShared";
+export { assert } from "./assert";
+export {
+    Buffer,
+    bufferToString,
+    gitHashFile,
+    hashFile,
+    IsoBuffer,
+    performance,
+    stringToBuffer,
+    Uint8ArrayToString,
+} from "./indexNode";
+export { fromBase64ToUtf8, fromUtf8ToBase64, toUtf8 } from "./base64Encoding";
```

Comment:
Would be good to sort files as well (base64Encoding before indexNode, etc.) - I know this issue preexists for this file, but as long as we're touching these files would be good to fix up.